tests/cli-misc.rs

```diff
@@ -785,15 +785,52 @@ fn completion_cargo_unsupported_shell() {
 }
 
 #[test]
-fn profiles() {
+fn set_profile() {
     setup(&|config| {
         expect_stdout_ok(config, &["rustup", "show", "profile"], "default");
+        assert!(config.exedir.join(format!("rls{}", EXE_SUFFIX)).exists());
         expect_ok(config, &["rustup", "set", "profile", "minimal"]);
         expect_stdout_ok(config, &["rustup", "show", "profile"], "minimal");
+        assert!(!config.exedir.join(format!("rls{}", EXE_SUFFIX)).exists());
```

Comment:
Why would the rls binary cease to exist at this point?

Reply:
because it is not part of the minimal profile (and was not explicitly added). However, I realise that the proxy will probably still be there.

Reply:
Proxies are always present because they might be used by different toolchains.  I guess these tests need reworking.